kalite/topic_tools/__init__.py

```diff
@@ -523,6 +518,29 @@ def get_assessment_item_data(request, assessment_item_id=None):
     return assessment_item
 
 
+def smart_translate_item_data(item_data):
+    """Auto translate the content fields of a given assessment item data.
+
+    An assessment item doesn't have the same fields; they change
+    depending on the question. Instead of manually specifying the
+    fields to translate, this function loops over all the relevant
+    fields of item_data and translates only the content field.
+    """
+    if 'content' in item_data:
+        item_data['content'] = _(item_data['content'])
+
+    for field in ASSESSMENT_DATA_TRANSLATABLE_FIELDS:
```

Comment:
>> Is it much more expensive to consider all of the keys rather than a limited subset?

Obviously if there are a lot more than 7 keys the answer is yes, but is there something else I'm not considering?